config/chime_science_run_recv.yaml

```diff
@@ -28,6 +28,10 @@ num_ev: 4
 # other stages (11 and 14, twinned with 3 and 6 respectively)
 cpu_affinity: [7, 11, 14, 15]
 
+telescope:
+  name: chime
+  allow_default_map: false
```

Comment:
The parameter name is a bit ambiguous. `allow_default_frequency_map` would be better, at the expense of being absurdly long. Your call!

Reply:
Changed.